### ISSUE TITLE
openBinaryFile: does not exist

### ISSUE THREAD

**Issue by VP59m** [qualiacode/arabica#7]
Hi Martin,
thank you so much for your last commit :)
When I executed pandoc with the options in arabica wiki, I had the following error message 
```
kpathsea:make_tex: Invalid filename \'Libertinus Serif\', contains \' \'
kpathsea:make_tex: Invalid filename \'Libertinus Serif\', contains \' \'
kpathsea:make_tex: Invalid filename \'Libertinus Serif\', contains \' \'
Error producing PDF.
! Package fontspec Error: The font "Libertinus Serif" cannot be found.

For immediate help type H <return>.
 ...                                              
                                                  
l.108     \setsansfont
```

Deleting Libertinus Serif/Sans/Mono/Math in 00_00_setting.yaml seems to resolve it, but I have onother error message  
`pandoc: ./output/projectname.pdf: openBinaryFile: does not exist (No such file or directory)`

How can I resolve?

regards Igor

**Comment by VP59m** [comment]
Ok creating the folder output into the project folder seems to fix it! :)

**Comment by VP59m** [comment]
Any ideas about Libertinus Serif error? I worked around the problem :). Can we talk here or it's better in an other issue? 

Regards Igor

**Comment by periodicpoint** [comment]
@VP59m, regarding your Libertinus font issue:

# BLUF
1. Go to [the Libertinus font git repository](https://github.com/libertinus-fonts/libertinus) and install the Libertinus font system-wide.
2. Alternatively install the _libertinus package_ with the package manager of your LaTeX distribution (e.g. 
 [TeX Live](https://tug.org/texlive/) or [MiKTeX](https://miktex.org/)); see also [CTAN](https://ctan.org/pkg/libertinus?lang=en).
3. Done. This should solve your issue.

# TL;DR
## Cause
1. The Libertinus font issue you are describing is most likely **not** an issue of _arabica_ itself.
2. Your issue is most likely **caused** by 
    1. **the absence of the Libertinus font** in your operating system **or**
    2. **the inaccessibility of the Libertinus font** to your LaTeX typesetting engine (eg. [XeLaTeX](https://en.wikipedia.org/wiki/XeTeX)).

## Explanation
1. In the settings file `00_00_setting.yaml ` at line 129-132 we are setting the main document font to Libertinus explicitly with:
`mainfont: 'Libertinus Serif',
sansfont: 'Libertinus Sans',
monofont: 'Libertinus Mono',
mathfont: 'Libertinus Math'`.
2. By calling pandoc with the XeLaTeX engine and these settings, XeLaTeX (if I assume correctly) looks up the main font, sans font, mono font and the math font respectively in the **texmf tree* e.g. ` /texmf-dist/fonts` (under Linux) or in your operating systems font directory e.g. `/usr/share/fonts/` (under Linux).
3. If XeLaTeX does not find those corresponding font files XeLaTeX will give you an error.

## Fix
See BLUF above.

Cheers

**Comment by VP59m** [comment]
Hi @qualiacode!
Thank you so much! I thought Libertinus font is in texlive-fontsextra, but maybe there is only Libertinus math font! Now installing in my Arch-based distro (Arcolinux) otf-libertinus-git, it's solved!

Thanks!!!
Cheers
